go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/open-policy-agent/opa
 
-go 1.20
+go 1.21
+
+toolchain go1.22.2
```

Comment:
This is probably your local Go version?

Reply:
Also this means we are increasing the min Go version to build OPA.

Reply:
Hmm it is my local version. I'll try dropping it.